schema/daqconf/triggergen.jsonnet

```diff
@@ -79,13 +79,20 @@ local cs = {
     # trigger options
     s.field( "completeness_tolerance", types.count, default=1, doc="Maximum number of inactive queues we will tolerate."),
     s.field( "tolerate_incompleteness", types.flag, default=false, doc="Flag to tell trigger to tolerate inactive queues."),
-    s.field( "ttcm_input_map", self.hsi_input_map, default=[
+    s.field( "ttcm_dts_input_map", self.hsi_input_map, default=[
       {"signal":0, "tc_type_name":"kTiming", "time_before":1000, "time_after":1000},
       {"signal":1, "tc_type_name":"kTiming", "time_before":1000, "time_after":1000},
       {"signal":2, "tc_type_name":"kTiming", "time_before":1000, "time_after":1000},
       {"signal":3, "tc_type_name":"kTiming", "time_before":1000, "time_after":1000}
```

Comment:
These lines will need the `"time_before"` and `"time_after"` removed after my more recent PR